WordPress/src/test/java/org/wordpress/android/sharedlogin/resolver/SharedLoginResolverTest.kt

```diff
@@ -18,19 +18,21 @@ import org.wordpress.android.fluxc.Dispatcher
 import org.wordpress.android.fluxc.annotations.action.Action
 import org.wordpress.android.fluxc.store.AccountStore
 import org.wordpress.android.fluxc.store.AccountStore.UpdateTokenPayload
+import org.wordpress.android.localcontentmigration.LocalMigrationContentProvider
+import org.wordpress.android.localcontentmigration.LocalMigrationContentResolver
 import org.wordpress.android.provider.query.QueryResult
 import org.wordpress.android.reader.savedposts.resolver.ReaderSavedPostsResolver
 import org.wordpress.android.resolver.ContentResolverWrapper
 import org.wordpress.android.sharedlogin.JetpackSharedLoginFlag
 import org.wordpress.android.sharedlogin.SharedLoginAnalyticsTracker
 import org.wordpress.android.sharedlogin.SharedLoginAnalyticsTracker.ErrorType
 import org.wordpress.android.util.publicdata.WordPressPublicData
-import org.wordpress.android.sharedlogin.provider.SharedLoginProvider
 import org.wordpress.android.ui.prefs.AppPrefsWrapper
 import org.wordpress.android.userflags.resolver.UserFlagsResolver
 import org.wordpress.android.util.AccountActionBuilderWrapper
 import org.wordpress.android.viewmodel.ContextProvider
 
+// TODO: adapt these tests to the unified provider / orchestrator approach
```

Comment:
Should we handle this before merging or in a different PR?

Reply:
Good question.. maybe we could try tackling these in this PR? The rationale being that our tests can help identify issues worth addressing in the implementation, and once it is more solid, later PRs based on this will have a stronger foundation. Wdyt?

Reply:
I had a second thought about this, and maybe this makes more sense after the other resolver / provider pairs are unified. My thinking is, by then, the error classes which dictate flow control will be more clearly defined, so we'll have a better idea of what to test for the orchestrator. Wdyt?